R/extract_random_variances.R

```diff
@@ -380,6 +380,12 @@ as.data.frame.VarCorr.lme <- function(x, row.names = NULL, optional = FALSE, ...
       # lme4 - wald / normal CI
 
       merDeriv_loaded <- isNamespaceLoaded("merDeriv")
+      # detach on exit
+      on.exit({
+        if (!merDeriv_loaded) {
```

Comment:
It's best-practice to also set `add = TRUE` and `after = FALSE` in `on.exit()` calls ([ref](https://adv-r.hadley.nz/functions.html?q=on.exit#on-exit)). I will change this directly in the main.